### PR TITLE
Version 1.1.0

### PR DESCRIPTION
**Pull request by Rocketct**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Arduino_MKRENV ?.?.? - ????.??.??
+
+Arduino_MKRENV 1.1.0 - 2019.05.31
+
+* Added imperial units support
+* Added readIlluminace() API to replace readLux()
```
Comment (sandeepmistry) [inline CHANGELOG]: There is a small typo here: `readIlluminace()` -> `readIlluminance()`

File: CHANGELOG [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Arduino_MKRENV ?.?.? - ????.??.??
+
+Arduino_MKRENV 1.1.0 - 2019.05.31
```
Comment (sandeepmistry) [inline CHANGELOG]: Just change to today's date, and we should be good :)

File: CHANGELOG (hunk 1 above)
Comment (Rocketct) [reply CHANGELOG]: done

File: CHANGELOG (hunk 2 above)
Comment (Rocketct) [reply CHANGELOG]: done

**Review by sandeepmistry — APPROVED** [review]

**Pull request merged**